CWorld.cpp

```diff
@@ -3,10 +3,11 @@
 //
 
 #include "CWorld.h"
-#include <string.h>
+#include <cstring>
 
```

Comment:
i do not know if this is relevant for us? those changes came with the default clang-tidy setup from clion

Reply:
cstring ist the STL version of C's string.h so they are different. In the long term string should be replaced with something else anyways, because it is one of the most stupid string implementations on earth. We use it for now to get the ball rolling.

Reply:
i say we should revert this setting and not let the linter mess with those imports